app/code/Magento/Swatches/Model/SwatchAttributeFrontendInputConversion.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Swatches\Model;
+
+/**
+ * Performs the conversion of the frontend input value for attribute data
+ */
+class SwatchAttributeFrontendInputConversion
```

Comment:
If we are talking about service, I would rename this class. So, the good practice is to start a service name with a verb. By following this practice we will get something like  `ConvertSwatchAttributeFrontendInput`. What do you think?

Reply:
You are right, I renamed the class.
The reason why I used such name - I tried to keep the same class name convent (if it's exists) for the module, like: SwatchAttributeCodes, SwatchAttributesProvider, SwatchAttributeType.